Makefile

```diff
@@ -85,6 +95,9 @@ windows-x64.test: windows-x64 test/run.py
 	$(DOCKER) run --rm dockcross/windows-x64 > $(BIN)/dockcross-windows-x64 && chmod +x $(BIN)/dockcross-windows-x64
 	$(BIN)/dockcross-windows-x64 python test/run.py --exe-suffix ".exe"
 
+Dockerfile: Dockerfile.in common.docker
+	cpp -o Dockerfile Dockerfile.in
```

Comment:
To keep syntax of Docker file simpler, would it make sense to use string substitution instead ? (using sed -e or similar)